src/AllocateParent.js

```diff
@@ -152,21 +122,19 @@ class AllocateParent extends Component {
   }
 
   render () {
-    if (this.state.error) {
+    if (this.props.error) {
       return (<div className="error-summary">
         <div className="error-message">
-          <div> {this.state.error} </div>
+          <div> {this.props.error} </div>
         </div>
       </div>);
     }
-    switch (this.state.page) {
+    switch (this.props.page) {
       case 1:
-        return <Unallocated list={this.state.list} gotoNext={this.gotoManualAllocation} {...this.props} />;
+        return <Unallocated gotoNext={this.gotoManualAllocation} {...this.props} />;
       case 2:
-        return (<ManualAllocation displayDateFilter allocatedKeyworkers={this.state.allocatedKeyworkers}
-          allocatedList={this.state.allocatedList} keyworkerList={this.state.keyworkerList}
-          handleKeyworkerChange={this.handleKeyworkerChange} toDate={this.state.toDate} fromDate={this.state.fromDate}
-          postManualOverride={this.postManualOverride} applyDateFilter={this.applyDateFilter} handleDateFilterChange={this.handleDateFilterChange} {...this.props} />);
```

Comment:
a lot of the explicit state passing disappears as redux has mapped it to this.props